config.sample.yaml

```diff
@@ -58,6 +58,9 @@ ircService:
       ssl: true
       # Whether or not IRC server is using a self-signed cert or not providing CA Chain
       sslselfsign: false
+      # Whether to allow expired certs when connecting to the IRC server.
+      # Usually this should be off. Default: false.
```

Comment:
a comment about how this interacts with/against `sslselfsign` would be nice.

Reply:
I think I'm failing to see how they intersect. This is about expired certs, not ones that might be invalid due to self signing?

Reply:
because I'm an idiot and read the wrong words. Ignore me on this comment - LGTM overall.

Reply:
:) *pats travis*